php/ext/google/protobuf/php-upb.h

```diff
@@ -868,7 +868,7 @@ bool upb_strtable_resize(upb_strtable *t, size_t size_lg2, upb_alloc *a);
  *    the program accesses an invalidated iterator.
  *
  * For C++11 range-based for:
- *  - iterators must be copyable
```

Comment:
I think in the C++ the "copyable" spelling is more common.

Reply:
Done